drivers/hmis/lib/form_data/allegheny/fragments/client_mci.json

```diff
@@ -38,7 +37,7 @@
         },
         {
           "question": "first_name",
-          "_comment": "dummy dependency to enable reset when field changes",
+          "_comment": "dummy dependency to enable reset when field changes. FIXME: first_name is not a valid reference; it doesn't exist in the form",
```

Comment:
we should get rid of these and have the MCI component handle it manually. It will be easier once we get react-hook-forms merged